### DISCUSSION TITLE
Custom login page title

### DISCUSSION THREAD

**Discussion by mxmeeple** [authelia/authelia#6555] [Ideas]
[ Continuing discussion from [Customize login page text](https://github.com/authelia/authelia/discussions/4671#discussioncomment-8142914) ]
Personally I like to have my SSO domain to share common titles, eg "Files - Domain", or "Login - Domain"
At the moment Authelia doesn't support this (yet?), I currently am using a rewrite Traefik middleware but I would much rather it be part of Authelia rather than hacked in. 
I believe this can me confusing for end users, and possibly cause them to think they're being misled by bad actors. Being redirected from "Files - Domain" to "Login - Authelia", especially with OIDC, can feel like you are leaving the domain and getting authenticated by Authelia (as on OIDC, not a Domain service), It could even feel like a MIM or Phishing. 
Personally I generally keep an eye on the URL and title, if they mismatch it gives me red flags.
This doesnt decrease security but could cause end users to simply not use the service feeling confusing/untrustworthy, contacting support for advice ("is this the right page?") etc

**Comment by subract** [comment]
Would love to see this as well. For my case, I'd like it for my password manager (which inspects the window/tab title to autotype the correct entry) to be able to distinguish between Authelia instances.

**Reply by james-d-elliott** [reply]
Which password manager? Are you sure it doesn't utilize `window.location.origin` (i.e. the origin/hostname) or `window.location` (i.e. the URL)?

**Reply by subract** [reply]
KeePassXC. I could definitely leverage its browser extension to give it access to the URL (or run an extension [like this one](https://addons.mozilla.org/en-US/firefox/addon/add-url-to-window-title/) to add the URL to the tile), but I prefer to keep my browser extensions minimal. 

It's definitely not an intractable problem, but it would be nice to be able to change the title.

**Reply by james-d-elliott** [reply]
Yeah we're willing to add the option however not for this reason. Using the page title is very insecure. I'd recommend their [official integration](https://keepassxc.org/docs/KeePassXC_GettingStarted#_install_the_browser_extension).

**Reply by subract** [reply]
I was initially upset with your critique of my approach, but you're right - I didn't consider how using the Autotype feature based on the window title opens me up to phishing attacks. I installed the extension and am working through adding URLs to my (hundreds of) accounts. Just wanted to drop by and say thanks!

**Comment by GeistFighter** [comment]
Hello, so there's currently no way of configuring the title differently?

**Reply by nherbaut** [reply]
I've used a [traefik plugin](https://plugins.traefik.io/plugins/62947307108ecc83915d7783/rewrite-body) to rewrite things based on regexp, so I can change the title and whatever i want on the login page.

**Reply by lokomass** [reply]
This is a workaround.
Is there a way to change the title text and the button text independently, even though they use the same label in the portal.json?
